### PR TITLE
Fix opencr kubernetes

### PR DESCRIPTION
**Pull request by rcrichton**

### CODE REVIEW

File: client/kubernetes/main/es-deployment.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   annotations:
-    kompose.cmd: kompose -f docker-compose.cicd.yml convert
-    kompose.version: 1.22.0 (HEAD)
+    kompose.cmd: kompose convert -f /home/ryan/git/instant/client/docker/docker-compose.yml
```
Comment (MattyJ007) [inline client/kubernetes/main/es-deployment.yaml:5]: This seems specific to your PC?

File: client/kubernetes/main/es-deployment.yaml [hunk 2]
```diff
@@ -18,12 +18,15 @@ spec:
   template:
     metadata:
       annotations:
-        kompose.cmd: kompose -f docker-compose.cicd.yml convert
-        kompose.version: 1.22.0 (HEAD)
+        kompose.cmd: kompose convert -f /home/ryan/git/instant/client/docker/docker-compose.yml
```
Comment (MattyJ007) [inline client/kubernetes/main/es-deployment.yaml:21]: Same here

File: client/kubernetes/main/es-service.yaml [hunk 3]
```diff
@@ -2,8 +2,8 @@ apiVersion: v1
 kind: Service
 metadata:
   annotations:
-    kompose.cmd: kompose -f docker-compose.cicd.yml convert
-    kompose.version: 1.22.0 (HEAD)
+    kompose.cmd: kompose convert -f /home/ryan/git/instant/client/docker/docker-compose.yml
```
Comment (MattyJ007) [inline client/kubernetes/main/es-service.yaml:5]: Same as above

File: client/kubernetes/main/opencr-deployment.yaml [hunk 4]
```diff
@@ -2,8 +2,8 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   annotations:
-    kompose.cmd: kompose -f docker-compose.cicd.yml convert
-    kompose.version: 1.22.0 (HEAD)
+    kompose.cmd: kompose convert -f /home/ryan/git/instant/client/docker/docker-compose.yml
```
Comment (MattyJ007) [inline client/kubernetes/main/opencr-deployment.yaml:5]: Same as above

File: client/kubernetes/main/opencr-deployment.yaml [hunk 5]
```diff
@@ -13,35 +13,47 @@ spec:
...
       io.kompose.service: opencr
-  strategy: {}
+  strategy:
+    type: Recreate
   template:
     metadata:
       annotations:
-        kompose.cmd: kompose -f docker-compose.cicd.yml convert
-        kompose.version: 1.22.0 (HEAD)
+        kompose.cmd: kompose convert -f /home/ryan/git/instant/client/docker/docker-compose.yml
```
Comment (MattyJ007) [inline client/kubernetes/main/opencr-deployment.yaml:21]: Same as above

File: client/kubernetes/main/opencr-service.yaml [hunk 6]
```diff
@@ -2,20 +2,19 @@ apiVersion: v1
 kind: Service
 metadata:
   annotations:
-    kompose.cmd: kompose -f docker-compose.cicd.yml convert
-    kompose.version: 1.22.0 (HEAD)
+    kompose.cmd: kompose convert -f /home/ryan/git/instant/client/docker/docker-compose.yml
```
Comment (MattyJ007) [inline client/kubernetes/main/opencr-service.yaml:5]: Same as above

File: core/kubernetes/main/k8s.sh [hunk 7]
```diff
@@ -144,8 +144,11 @@ elif [ "$1" == "down" ]; then
     kubectl delete deployment hapi-fhir-server-deployment
     kubectl delete deployment hapi-fhir-mysql-deployment
 elif [ "$1" == "destroy" ]; then
-    kubectl delete deployment,service,statefulset,pvc,pv,job -l package=core
-    kubectl delete configmap --all
+    kubectl delete -f $k8sMainRootFilePath/mongo/mongo-volume.yaml
```
Comment (MattyJ007) [inline core/kubernetes/main/k8s.sh:147]: Perfecto :D

**Review by MattyJ007 — COMMENTED** [review]
MInor path thing

**Comment by MattyJ007** [comment]
If you have time perhaps you could add the URL in the k8s script to output the OpenCR local URL? Very helpful to get started

**Comment by rcrichton** [comment]
@MattyJ007 about the URL scripts, I want to rethink those sometime. I found a command `minikube tunnel` which makes minikube behave like a cloud service. I.e. it allows external IPs to be assigned to LoadBalancers. So, I don't want to add any more code around those as we might want to redo it.

**Pull request merged**